src/Components/Admin/admin.js

```diff
@@ -279,6 +287,25 @@ export default function Admin() {
         setUploadAssetModalOpen(false);
     };
 
+    const handleDeleteAssetClick = (assetId) => {
+        setDeleteAssetId(assetId);
+        setDeleteAssetModalOpen(true);
+    };
+
+    const handleDeleteAssetCancel = () => {
+        setDeleteAssetId(null);
+        setDeleteAssetModalOpen(false);
+    };
+
+    const handleDeleteAssetSubmit = async () => {
+        await deleteAsset(deleteAssetId, handleError);
+
+        const modifiedAssets = assets.filter((env) => env.id !== deleteAssetId);
```

Comment:
Nit: the iterated object variable name in the filter shouldn't be `env`. Can we do `asset` instead?

Reply:
Thanks, updated :)